src/components/Characters/Characters.test.jsx

```diff
@@ -0,0 +1,28 @@
+import { render } from '@testing-library/react';
+import { Link } from 'react-router-dom';
+import { MemoryRouter } from 'react-router-dom/cjs/react-router-dom.min';
+import Characters from './Characters';
+
+describe('characters display', () => {
+  it('renders character list', () => {
+    const mock = [
```

Comment:
this should be just a basic array with objects, like: `[{id: 1, name: 'Rick', image: 'www.rick.com'},{id: 2, name: 'Morty', image: 'www.morty.com'}]`